dashboard/src/components/staff/Staff.vue

```diff
@@ -59,8 +59,10 @@ export default {
 </script>
 
 <style scoped>
-.container {
-	margin: 2em;
+.Staff {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
 }
 #staff {
```

Comment:
This could be made clearer. Is `#staff` referenced in code? If not it can be turned into a class named stafflist.

Reply:
should probably be a class